domain/consensus/utils/consensushashing/calculate_signature_hash.go

```diff
@@ -14,16 +14,26 @@ type SigHashType uint32
 
 // Hash type bits from the end of a signature.
 const (
-	SigHashAll          SigHashType = 0x1
-	SigHashNone         SigHashType = 0x2
-	SigHashSingle       SigHashType = 0x3
-	SigHashAnyOneCanPay SigHashType = 0x80
+	SigHashAll          SigHashType = 0b00000001
+	SigHashNone         SigHashType = 0b00000010
+	SigHashSingle       SigHashType = 0b00000100
```

Comment:
That's 4, it should be 3

Reply:
This is deliberate.
@stasatdaglabs asked me to do this so that SigHashType is a true bitfield.

LMK if you disagree